segments/gcalcli.sh

```diff
@@ -0,0 +1,32 @@
+# Print next Google Calendar Event
+
+TMUX_POWERLINE_SEG_GCALCLI_MILITARY_TIME_DEFAULT=1
+
+generate_segmentrc() {
+	read -d '' rccontents  << EORC
+gcalcli uses military time by default - if you want to see 12 hour time format, set TMUX_POWERLINE_SEG_GCALCLI_MILITARY_TIME_DEFAULT to 0
```

Comment:
Prepend ‘#’ to make this a comment, the generated file is a shell script:)

Reply:
Ah thanks for catching that! Totally missed that.